README.md

```diff
@@ -128,7 +128,7 @@ bin/moodle-docker-compose exec webserver php admin/cli/install_database.php --ag
 
 Notes:
 * Moodle is configured to listen on `http://localhost:8000/`.
-* Mailhog is listening on `http://localhost:8000/_/mail` to view emails which Moodle has sent out.
+* Mailpit is listening on `http://localhost:8025/_/mail` to view emails which Moodle has sent out.
```

Comment:
```suggestion
* Mailpit is listening on `http://localhost:8000/_/mail` to view emails which Moodle has sent out.
```

I think `8000` is the correct public address that apache is proxying from internal (not published) `8025`.

Reply:
Ahh, thanks for the pickup. That is a leftover from an earlier version of the patch. I have fixed it now and force pushed

Cheers,
Matt P